packages/extensions/openapi-to-typespec/test/arm-apimanagement/tsp-output/ApiContract.tsp

```diff
@@ -46,7 +46,6 @@ interface ApiContracts {
    */
   #suppress "@azure-tools/typespec-azure-core/no-openapi" "non-standard operations"
   #suppress "@azure-tools/typespec-azure-resource-manager/arm-put-operation-response-codes" "For backward compatibility"
-  #suppress "@azure-tools/typespec-azure-resource-manager/no-response-body" "For backward compatibility"
   @operationId("Api_CreateOrUpdate")
```

Comment:
This is Pt. 3 I mentioned in the description. We don't need the suppression here.